dbt/adapters/bigquery/relation_configs/_options.py

```diff
@@ -22,6 +23,7 @@ class BigQueryOptionsConfig(BigQueryBaseRelationConfig):
     refresh_interval_minutes: Optional[float] = 30
     expiration_timestamp: Optional[datetime] = None
     max_staleness: Optional[str] = None
+    allow_non_incremental_definition: Optional[bool] = False
```

Comment:
This should be `None`, so that it drops out of the config dict if it's not set. It also looks like there is no default according to the [docs](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-definition-language#materialized_view_option_list). It's an odd way of saying it, but they say `allow_non_incremental_definition=true is disabled by default` versus `Default: true` as appears elsewhere.